File: src/azk/cli/detect/ruby.lua
local utils = require('azk.utils')
local fs    = require('azk.utils.fs')

local ruby = {}
local box  = {
  box = "azukiapp/ruby-box#stable"
}

function ruby.detect(path)
  if fs.is_regular(path .. "/" .. "Gemfile") then
    return box
  end
end

return ruby

File: lib/detector/detectors/ruby.js
var path = require('path');
var fs   = require('fs');

var node = {}
var box  = {
  box: "azukiapp/ruby-box#stable"
}

module.exports = {
  detect: function(dir) {
    var file = path.join(dir, "Gemfile")
    if (fs.existsSync(file)) {
      return box;
    }
  }
}

